source/installation-guide/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst

```diff
@@ -83,7 +83,7 @@ Elasticsearch installation and configuration
 Certificates creation and deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. [*Master*] This step implies the selection of the Wazuh cluster installation type. In case of configuring a Wazuh single node cluster, the ``Wazuh single-node cluster`` tab must be selected. In case of having two or more nodes, the ``Wazuh multi-node cluster`` tab must be choosen.
+#. [*Master*] This step implies the selection of the Wazuh cluster installation type. Choose between ``Wazuh single-node cluster`` and ``Wazuh multi-node cluster`` tab depending on your intallation.
```

Comment:
Maybe we need to specify what is the difference between both installations. Somebody who doesn't know the difference between Wazuh single-node and multi-node could be grateful if we offer here a little help.